From b1c1b44ea2c9da008ee7d23864b7f28026ff5a3a Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Wed, 9 Aug 2017 12:28:39 -0700
Subject: [PATCH] MANTA-3369 update cueball to 2.2.9 to help debug long
 requests Reviewed by: Dave Pacheco <dap@joyent.com> Reviewed by: Jordan
 Hendricks <jordan.hendricks@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 160f8d8..4c9e8a7 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "backoff": "2.3.0",
         "bunyan": "0.22.1",
         "bunyan-syslog": "0.2.2",
-        "cueball": "2.2.8",
+        "cueball": "2.2.9",
         "dashdash": "1.3.2",
         "deep-equal": "0.0.0",
         "dtrace-provider": "0.2.8",
-- 
2.21.0

